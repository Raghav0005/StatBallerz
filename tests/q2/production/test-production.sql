SELECT GameID, avg FROM 
Games g JOIN (
    SELECT GameID as gid, AVG(TotalRebounds) as avg
    FROM PlayedIn
    GROUP BY GameID
) ON g.GameID = gid
WHERE g.GameDate >= '2024-11-01' AND g.GameDate <= '2024-11-05';