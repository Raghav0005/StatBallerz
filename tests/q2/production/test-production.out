
GAMEID      AVG        
----------- -----------

  0 record(s) selected.


